pro/src/screens/Bookings/PreFilters/PreFilters.tsx

```diff
@@ -8,9 +8,12 @@ import { DEFAULT_PRE_FILTERS, TPreFilters } from 'core/Bookings'
 import { GetBookingsCSVFileAdapter } from 'core/Bookings'
 import { ReactComponent as ResetIcon } from 'icons/reset.svg'
 import FilterByOfferType from 'new_components/FilterByOfferType'
+import MultiDownloadButtonsModal from 'new_components/MultiDownloadButtonsModal/MultiDownloadButtonsModal'
 import { Button } from 'ui-kit'
 import { ButtonVariant } from 'ui-kit/Button/types'
 
+import getBookingsXLSFileAdapter from '../../../routes/Bookings/adapters/getBookingsXLSFileAdapter'
+
```

Comment:
```suggestion
import getBookingsXLSFileAdapter from 'routes/Bookings/adapters/getBookingsXLSFileAdapter'
```

Le screen n'est pas sensé importe depuis la route.
les methods `getBookingsXLSFileAdapter` et `getBookingsXLSFileAdapter` devrait etre utilisé dans la route et `downloadBookingsCSV` passé en props.
Ici je vois bien que ca cloche car on conditionne le `disabled` du bouton à l'état de la requete.

Je pense qu'on pourrai bouger cette adapter dans le composant `screens/Bookings/PreFilters/ButtonDropdownDownalod` que je proposait de créer plus haut.

@mariedestandau , @MathildeDuboille un vous en pensez quoi ?

Reply:
Le plus simple de l'importer directement, mais ce n'est pas cohérent avec la nouvelle archi. 
On pourrait laisser comme ça (importé directement), et ça pourrait être l'occasion de partir sur une règle plus flexible du type: 
- si l'adapteur change en fonction de la route, on le passe en prop dans la route
- si l'adapteur est toujours le même, on l'importe là où il est utilisé
Qu'est ce que vous en pensez ?